src/topology/continuous_function/basic.lean

```diff
@@ -58,8 +58,8 @@ lemma ext_iff : f = g ↔ ∀ x, f x = g x :=
 instance [inhabited β] : inhabited C(α, β) :=
 ⟨{ to_fun := λ _, default, }⟩
 
-lemma coe_inj ⦃f g : C(α, β)⦄ (h : (f : α → β) = g) : f = g :=
-by cases f; cases g; cases h; refl
+lemma coe_injective : @function.injective (C(α, β)) (α → β) coe_fn :=
+λ f g h, by cases f; cases g; congr'
```

Comment:
I know this should use `fun_like`, but I didn't want to do too many refactors at the same time.